book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :tittle, :author
+  attr_reader :rentals
```

Comment:
- [ ] You have a typo in line 2, It's supposed to be `:title` instead of `:tittle` so please fix that.
- [ ] Please add Getters and Setters for `@rentals` instead of getters only So the book class can add(Set) a new rental.